src/Console/Command/DescribeCommand.php

```diff
@@ -174,14 +174,17 @@ private function describeRule(OutputInterface $output, $name)
                 }
 
                 $description = preg_replace('/(`.+?`)/', '<info>$1</info>', $option->getDescription());
-                $line .= ': '.lcfirst(preg_replace('/\.$/', '', $description)).'; ';
```

Comment:
I moved it because now there is one case where `;` is not needed: when the default value is computed at runtime with a callable.